vulture/core.py

```diff
@@ -88,6 +88,15 @@ def read_file(filename):
         raise VultureInputException(err)
 
 
+def _get_suitable_whitelists(defined_imports):
+    path = os.path.dirname(os.path.abspath(__file__))
+    whitelist_dir = os.path.join(path, 'whitelists')
+    all_whitelists = os.listdir(whitelist_dir)
```

Comment:
Is there any other way through which we can enlist the `package_data`? 
(There is nothing as such in the documentation) :(